otaclient/app/boot_control/_cboot.py

```diff
@@ -260,6 +262,7 @@ def switch_boot(self):
         slot = self.standby_slot
 
         logger.info(f"switch boot to {slot=}")
+        Nvbootctrl.set_active_boot_slot(slot, target="bootloader")
```

Comment:
nvbootctrl with bootloader support

Reply:
So if we specify `nvbootctrl -t bootloader`, and select the next boot slot to slot a(0), the device will not only boots with rootfs in slot a(0), then it will also use bootloader and firmwares for slot a, is that correct?

Reply:
So if we specify `nvbootctrl -t bootloader`, and select the next boot slot to slot a(0), the device will not only boots with rootfs in slot a(0), then it will also use bootloader and firmwares for slot a, is that correct?

Reply:
So if we specify `nvbootctrl -t bootloader`, and select the next boot slot to slot a(0), the device will not only boots with rootfs in slot a(0), then it will also use bootloader and firmwares for slot a, is that correct?

Reply:
target rootfs and bootloader seems to be independent.
target=rootfs is for rootfs, and target=bootloader is for firmware.
If we specify rootfs active slot=0 and bootloader active slot=0, both boot from slot 0.